package.json

```diff
@@ -291,14 +292,14 @@
                     "when": "view == extension.vsKubernetesExplorer && viewItem == vsKubernetes.resource.pod"
                 },
                 {
-                    "command": "extension.vsKubernetesLogs",
+                    "command": "extension.vsKubernetesShowLogs",
                     "group": "3",
                     "when": "view == extension.vsKubernetesExplorer && viewItem == vsKubernetes.resource.pod"
                 },
                 {
-                    "command": "extension.vsKubernetesLoad",
```

Comment:
I think this may have been accidentally deleted.

Reply:
Sorry about that, I selected that line by mistake.